src/icons/Activity.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import colors from 'src/styles/colors'
+import Svg, { Path } from 'svgs'
+
+interface Props {
+  size?: number
+  color?: colors
+}
+
+function Activity({ color = colors.successDark, size = 24 }: Props) {
+  return (
+    <Svg width={size} height={size} fill="none">
```

Comment:
I believe we also need a `viewBox` property for [correct scaling](https://graphicdesign.stackexchange.com/a/71574).

```suggestion
    <Svg width={size} height={size} fill="none" viewBox="0 0 24 24">
```